README.rst

```diff
@@ -0,0 +1,78 @@
+About the Let's Encrypt Client
+==============================
+
+In short: getting and installing SSL/TLS certificates made easy.
+
+The Let's Encrypt Client is a tool that talks to the Let's Encrypt CA
+so you can comfortably and quickly get trusted TLS certificates that just
+work without warnings in every browser.
+
+It's all automated:
+
+* The tool will prove domain control to the CA and submit a CSR (Certificate
+  Signing Request).
+* If domain control has been proven, a certificate will get issued and the tool
+  will automatically install it.
+
+All you need to do is:
+
+::
+
+   user@www:~$ sudo letsencrypt www.example.org
+
+
+**Encrypt ALL the things!**
+
+
+.. image:: https://travis-ci.org/letsencrypt/lets-encrypt-preview.svg?branch=master
+    :target: https://travis-ci.org/letsencrypt/lets-encrypt-preview
+
+
+Disclaimer
+----------
+
+This is a **DEVELOPER PREVIEW** intended for developers and testers only.
+
+**DO NOT RUN THIS CODE ON A PRODUCTION SERVER. IT WILL INSTALL CERTIFICATES
+SIGNED BY A TEST CA, AND WILL CAUSE CERT WARNINGS FOR USERS.**
+
+
+Features
+========
+
+* web servers supported:
+
+  - apache2.x (tested and working on Ubuntu Linux)
+
+* the private key is generated locally on your system
+* can talk to the Let's Encrypt (demo) CA or optionally to other ACME
+  compliant services
+* can get domain-validated (DV) certificates
+* can revoke certificates
+* adjustable RSA key bitlength (2048 (default), 4096, ...)
+* optionally can install a http->https redirect, so your site effectively
+  runs https only
+* fully automated
+* configuration changes can be rolled back N checkpoints
+* text and ncurses UI
+* Free and Open Source Software, made with Python.
+
+
+Links
+-----
+
+Documentation: https://letsencrypt.readthedocs.org/
+
+Software project: https://github.com/letsencrypt/lets-encrypt-preview
+
+Main Website: https://letsencrypt.org/
+
+IRC Channel: #letsencrypt on `Freenode`_
+
+Mailing list: `client-dev`_ (to subscribe without a Google account, send an
```

Comment:
line above gets rendered (in GitHub) as ***strong italics***, line below does not